client/src/modules/employees/registry/search.modal.js

```diff
@@ -90,7 +91,7 @@ function EmployeeRegistryModalController(ModalInstance, bhConstants, moment, Ser
       if (angular.isDefined(value)) {
         // default to the original value if no display value is defined
         var displayValue = displayValues[key] || lastDisplayValues[key] || value;
-        changes.post({ key: key, value: value, displayValue: displayValue });
+        changes.post({ key: key, value: value, displayValue: value });
```

Comment:
Is this change really necessary?  It looks like it undoes the work by @lomamech on displaying human-readable values instead of IDs.

Reply:
I don't understand what is happening if I use  @lomamech  method here,  if I let  things like this
changes.post({ key: key, value: value, displayValue: displayValue });
the first store works but no the second.
When I try to change  doing things like this 
changes.post({ key: key, value: value, displayValue: value });
every thing works flawlessly

Reply:
Alright, I'll fork this PR and see if I can get it to work.  I don't want to destroy @lomamech's work.

Reply:
I have already used @lomamech's  storing style in another issue, it worked without any problem. But I don't know why it poses problem here